scene-lib/src/scenelib/annotations/field/AnnotationAFT.java

```diff
@@ -49,9 +51,36 @@ public  String toString() {
     /**
      * {@inheritDoc}
      */
+    // TODO: reduce duplication with annotator.specification.IndexFileSpecification.getElementAnnotations(AElement)
     @Override
     public String format(Object o) {
-        return o.toString();
+        Annotation anno = (Annotation) o;
```

Comment:
I suggest this code belongs in Annotation.toString().  There, it will address an existing comment, and it will be used in other contexts.